rosdep/base.yaml

```diff
@@ -1851,6 +1851,8 @@ hddtemp:
   slackware: [hddtemp]
   ubuntu:
     '*': null
+    trusty: [hddtemp]
+    xenial: [hddtemp]
     bionic: [hddtemp]
     focal: [hddtemp]
```

Comment:
Indeed Xenial and Trusty support has been dropped. We can't keep EOL distributions in here forever - they have to be retired at some point. You're always free to point rosdep at a custom database or a previous version of this database to get things working on those older platforms, but they're not supported at HEAD.

Even further, support for this package has been dropped from Ubuntu itself.

Reply:
OK, thanks